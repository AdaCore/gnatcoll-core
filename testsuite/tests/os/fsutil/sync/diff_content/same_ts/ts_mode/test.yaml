description: Test for GNATCOLL.OS.FSutil.Sync_Trees. Scenario with mode set to
             TIMESTAMP and with files having the same timestamps but different
             content