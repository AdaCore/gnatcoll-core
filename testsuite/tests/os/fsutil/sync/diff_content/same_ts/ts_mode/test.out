Test for GNATCOLL.OS.FSUtil.Sync_Trees
OK   test.adb:15
Source tree
===========
DIR src_top_dir/dir-0
DIR src_top_dir/dir-1
DIR src_top_dir/dir-2
FILE src_top_dir/dir-0/file-0
FILE src_top_dir/dir-0/file-1
FILE src_top_dir/dir-0/file-2
FILE src_top_dir/dir-1/file-0
FILE src_top_dir/dir-1/file-1
FILE src_top_dir/dir-1/file-2
FILE src_top_dir/dir-2/file-0
FILE src_top_dir/dir-2/file-1
FILE src_top_dir/dir-2/file-2
FILE src_top_dir/file-0
FILE src_top_dir/file-1
FILE src_top_dir/file-2

Destination tree
================
DIR dst_top_dir/dir-0
DIR dst_top_dir/dir-1
DIR dst_top_dir/dir-2
FILE dst_top_dir/dir-0/file-0
FILE dst_top_dir/dir-0/file-1
FILE dst_top_dir/dir-0/file-2
FILE dst_top_dir/dir-1/file-0
FILE dst_top_dir/dir-1/file-1
FILE dst_top_dir/dir-1/file-2
FILE dst_top_dir/dir-2/file-0
FILE dst_top_dir/dir-2/file-1
FILE dst_top_dir/dir-2/file-2
FILE dst_top_dir/file-0
FILE dst_top_dir/file-1
FILE dst_top_dir/file-2

Content checking
================
Content differs for the file dir-0/file-0
Content differs for the file dir-0/file-1
Content differs for the file dir-0/file-2
Content differs for the file dir-1/file-0
Content differs for the file dir-1/file-1
Content differs for the file dir-1/file-2
Content differs for the file dir-2/file-0
Content differs for the file dir-2/file-1
Content differs for the file dir-2/file-2
Content differs for the file file-0
Content differs for the file file-1
Content differs for the file file-2
<=== TEST PASSED ===>
